public/static/docs/api-reference/read.md

```diff
@@ -0,0 +1,103 @@
+# read()
+
+Returns the contents of a tracked file.
+
+```py
+dvc.api.open(path: str,
+             repo: str = None,
+             rev: str = None,
+             remote: str = None,
+             mode: str = "r",
+             encoding: str = None)
+```
+
+#### Usage:
+
+```py
+import dvc.api
+
+modelpkl = dvc.api.read(
+    'model.pkl',
+    repo='https://github.com/example/project.git'
+    mode='rb')
+```
+
+## Description
+
+This function wraps [`dvc.api.open()`](/doc/api-reference/open) for a simple and
+direct way to return the complete contents of files tracked in <abbr>DVC
+projects</abbr> (by DVC or Git). If the file cannot be found, a
+`PathMissingError` is raised.
```

Comment:
move to Exceptions?

Reply:
see also some comments to open() that could be relevant here as well

Reply:
Addressed.